frontend/packages/client/src/components/common/Toast.js

```diff
@@ -1,43 +1,38 @@
-import { useHistory } from 'react-router-dom';
 import { Svg } from '@cast/shared-components';
 
-const Toast = ({ onClose, text, footerText, footerLink }) => {
-  const history = useHistory();
-
-  return (
+const Toast = ({ history, onClose, text, footerText, footerLink }) => (
```

Comment:
Can we adjust the structure of this Toast component to be something like this:
```
<wrapperDiv>
    <closeButton/>
    {children}
</wrapperDiv>

```
Reason: this way we can pass any children components into the toast, have the business logic within their own context, and make the toast component more flexible and reusable